event-stream-processing/src/node-css-parser.test.ts

```diff
@@ -0,0 +1,149 @@
+import { NodeCssParser } from './node-css-parser';
+
+test('basic', () => {
+  const filter = new NodeCssParser();
+
+  /* eslint-disable */
+
+  // record from fluent-bit looks like this:
```

Comment:
I think you can just make a comment referring to the unit tests for an examples of records.

Reply:
thanks.. i dont think i need this commented code anymore because the incoming log matches the subject 'record' being used in the unit test.